python/ray/data/dataset.py

```diff
@@ -353,8 +353,9 @@ def map_batches(
             fn: The function to apply to each record batch, or a class type
                 that can be instantiated to create such a callable. Callable classes are
                 only supported for the actor compute strategy.
-            batch_size: Request a specific batch size, or None to use entire
-                blocks as batches. Defaults to a system-chosen batch size.
+            batch_size: Request a specific batch size, or None to use entire blocks
```

Comment:
Can we call out the size is in num of rows? I saw some confusion about 4096 being in num of bytes.

Reply:
ah yes, that would be great. e.g. [pyarrow also calls out the row count](https://github.com/apache/arrow/blob/master/python/pyarrow/_dataset.pyx#L2253-L2254).